install/kubernetes/cilium/templates/etcd-operator/poddisruptionbudget.yaml

```diff
@@ -7,7 +7,8 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     io.cilium/app: etcd-operator
-    name: cilium-etcd-operator
```

Comment:
For consistency, let's also keep the labels on the poddisruptionbudget here. I think we could remove them from all components going forward (as long as we also update the well-known security identities), but I'd like to do that in a separate PR, as I think that might be a bigger undertaking (I also see those labels being used e.g. in CI).

Reply:
@gandro Thanks!!  I miss this.